app/controllers/banners_controller.rb

```diff
@@ -65,7 +65,12 @@ def set_banner
   end
 
   def banner_params
-    params.require(:banner).permit(:active, :content, :name).merge(user: current_user)
+    params.require(:banner).permit(:active, :content, :name, :expires_at).merge(user: current_user)
+      .tap { |banner_params| set_expires_at_in_user_time_zone(banner_params) }
+  end
+
+  def set_expires_at_in_user_time_zone(banner_params)
+    banner_params[:expires_at] = banner_params[:expires_at].in_time_zone(cookies[:browser_time_zone])
```

Comment:
Is this method testable? Can we make it testable and then write some tests for it? Maybe this should be a method on the model.

Reply:
I believe the tests for `BannerParameters` cover this now. I debated putting it on the model originally but formatting parameters from a request seems like a controller responsibility, so I think having it in the controller/params object makes sense but let me know if you think otherwise